backend/app/modules/imagefetcher/FetcherActor.scala

```diff
@@ -70,26 +72,61 @@ class FetcherActor(ws: WSClient, serverUrl: String, monitoring: ActorRef) extend
       if (message.scale.isDefined) initialParams :+ ("scale", message.scale.get.toString)
       else initialParams
 
-    val request = ws.url(serverUrl + "/rgb")
+    fetchImage("/rgb", params, task, message.queryId)
+  }
+
+  /**
+    * Ask the flask server to generate an url to download a Forest Diff image
+    * @param message Contains all the information about the request
+    */
+  def fetchForestDiff(message: FetchForestDiff): Unit = {
+    val task = monitoring.ask(StartTask(message.queryId, "Fetching image"))(50.seconds).mapTo[Task]
+
+    val initialParams = Seq(
```

Comment:
[discussion] so we avoid parameters such as `city` or `country`? Should I keep them in case we use them later?